docs/how-to/hip_runtime_api/memory_management/unified_memory.rst

```diff
@@ -0,0 +1,737 @@
+
+Driver managed page migration
+--------------------------------------------------------------------------------
+
+If the hardware does not support on-demand page migration, then all the pages
+accessed by a kernel have to be resident on the device, so they have to be
+migrated before the kernel is running. Since the driver can not know beforehand,
+what parts of an array are going to be accessed, all pages of all accessed
+arrays have to be migrated. This can lead to significant delays on the first run
+of a kernel, on top of possibly copying more memory than is actually accessed by
+the kernel.
+
+.. _unified memory system requirements:
+
+System requirements
+================================================================================
+
+Unified memory is supported on Linux by all modern AMD GPUs from the Vega
+series onward. Unified memory management can be achieved with managed memory
+allocation and, for the latest GPUs, with a system allocator.
+
+The table below lists the supported allocators for the different architectures.
+The allocators are described in the next section.
+
+.. list-table:: Supported Unified Memory Allocators
+    :widths: 40, 25, 25
+    :header-rows: 1
+    :align: center
+
+    * - Architecture
+      - :cpp:func:`hipMallocManaged()`, ``__managed__``
+      - ``new``, ``malloc()``
+    * - CDNA3
+      - ✅
+      - ✅ :sup:`1`
+    * - CDNA2
+      - ✅
+      - ✅ :sup:`1`
+    * - CDNA1
+      - ✅
+      - ✅ :sup:`1`
+    * - RDNA1
+      - ✅
+      - ❌
+    * - GCN5
+      - ✅
+      - ❌
+
+✅: **Supported**
+
+❌: **Unsupported**
+
+:sup:`1` Works only with ``XNACK=1``. First GPU access causes recoverable
+page-fault. For more details, visit
+`GPU memory <https://rocm.docs.amd.com/en/latest/conceptual/gpu-memory.html#xnack>`_.
+
+.. _unified memory allocators:
+
+Unified memory allocators
+================================================================================
+
+Support for the different unified memory allocators depends on the GPU
+architecture and on the system. For more information, see :ref:`unified memory
+system requirements` and :ref:`checking unified memory management support`.
+
+- **HIP allocated managed memory and variables**
+  
+  :cpp:func:`hipMallocManaged()` is a dynamic memory allocator available on
+  all GPUs with unified memory support. For more details, visit
+  :ref:`unified_memory_reference`.
+
+  The ``__managed__`` declaration specifier, which serves as its counterpart,
+  can be utilized for static allocation.
+
+- **System allocated unified memory**
+
+  Starting with CDNA2, the ``new`` and `malloc()`` system allocators allow
+  you to reserve unified memory. The system allocator is more versatile and
+  offers an easy transition for code written for CPUs to HIP code as the
+  same system allocation API is used.
+
+To ensure the proper functioning of system allocated unified memory on supported
+graphics cards, it is essential to configure the environment variable
```

Comment:
```suggestion
accelerators and graphics cards, it is essential to configure the environment variable
```

Reply:
There was no distinction between GPUs and accelerators in this whole chapter, and I won't start here, especially as the differences are miniscule at best

Reply:
You differentiate between GPUs and graphic cards?

Reply:
I don't think anyone would really see the terms "GPU" and "graphics card" as differentiation, they are synonyms. Changed it either way.